.idea/compiler.xml

```diff
@@ -4,6 +4,8 @@
     <bytecodeTargetLevel>
       <module name="MadeInAbyss_main" target="1.8" />
       <module name="MadeInAbyss_test" target="1.8" />
+      <module name="MineInAbyss_main" target="1.8" />
```

Comment:
Oops this is my bad. I named my project MadeInAbyss for some stupid reason XD. Ill fix locally after merge